pkg/alertmanager/alertmanager.go

```diff
@@ -266,6 +268,7 @@ func New(cfg *Config, reg *prometheus.Registry) (*Alertmanager, error) {
 		GroupFunc: func(f1 func(*dispatch.Route) bool, f2 func(*types.Alert, time.Time) bool) (dispatch.AlertGroups, map[model.Fingerprint][]string) {
 			return am.dispatcher.Groups(f1, f2)
 		},
+		Concurrency: apiConcurrency,
```

Comment:
this won't respect the run-time configuration unless tenant re-created the am config

Reply:
Thank you for this insight. We are moving this back as a static config.